doc/en/source/system_overview.rst

```diff
@@ -111,6 +111,16 @@ to enable FogFlow Ecosystem to interact with Scorpio context broker. The NGSI-LD
 .. _`Integrate FogFlow with Scorpio Broker`: https://fogflow.readthedocs.io/en/latest/scorpioIntegration.html
 
 
+**FogFlow NGSI-LD Support** FogFlow is providing NGSI-LD API support along with NGSI9 and NGSI10. 
+NGSI-LD format aims at utilizing the **linked data model used by Fiware** for the purpose of deriving a hassle free and better mode of communication among components of fogflow or other GE's. It reduces the complexity of maintaining the data among NGSIv1 and NGSIv2 model by establishing a relationship between data to deduce information in a more efficient manner.
```

Comment:
[minor]NGSI-LD feature like unambiguous and better understanding of context sharing need to be added.

Reply:
done